examples/IBLevelSet/ex1/input2d_dynamic_mesh

```diff
@@ -0,0 +1,438 @@
+}
+
+VelocityBcCoefs_1 {
+   acoef_function_0 = "1.0"
+   acoef_function_1 = "1.0"
+   acoef_function_2 = "1.0"
+   acoef_function_3 = "1.0"
+
+   bcoef_function_0 = "0.0"
+   bcoef_function_1 = "0.0"
+   bcoef_function_2 = "0.0"
+   bcoef_function_3 = "0.0"
+
+   gcoef_function_0 = "0.0"
+   gcoef_function_1 = "0.0"
+   gcoef_function_2 = "0.0"
+   gcoef_function_3 = "0.0"
+}
+
+DensityBcCoefs {
+   acoef_function_0 = "0.0"
+   acoef_function_1 = "0.0"
+   acoef_function_2 = "0.0"
+   acoef_function_3 = "0.0"
+
+   bcoef_function_0 = "1.0"
+   bcoef_function_1 = "1.0"
+   bcoef_function_2 = "1.0"
+   bcoef_function_3 = "1.0"
+
+   gcoef_function_0 = "0.0"
+   gcoef_function_1 = "0.0"
+   gcoef_function_2 = "0.0"
+   gcoef_function_3 = "0.0"
+}
+
+ViscosityBcCoefs {
+   acoef_function_0 = "0.0"
+   acoef_function_1 = "0.0"
+   acoef_function_2 = "0.0"
+   acoef_function_3 = "0.0"
+
+   bcoef_function_0 = "1.0"
+   bcoef_function_1 = "1.0"
+   bcoef_function_2 = "1.0"
+   bcoef_function_3 = "1.0"
+
+   gcoef_function_0 = "0.0"
+   gcoef_function_1 = "0.0"
+   gcoef_function_2 = "0.0"
+   gcoef_function_3 = "0.0"
+}
+
+PhiBcCoefs {
+   acoef_function_0 = "0.0"
+   acoef_function_1 = "0.0"
+   acoef_function_2 = "0.0"
+   acoef_function_3 = "0.0"
+
+   bcoef_function_0 = "1.0"
+   bcoef_function_1 = "1.0"
+   bcoef_function_2 = "1.0"
+   bcoef_function_3 = "1.0"
+
+   gcoef_function_0 = "0.0"
+   gcoef_function_1 = "0.0"
+   gcoef_function_2 = "0.0"
+   gcoef_function_3 = "0.0"
+}
+
+// Simulation Parameters
+DELTA_FUNCTION       = "IB_6"
+SOLVER_TYPE          = "STAGGERED"
+START_TIME           =   0.0e0
+END_TIME             =   26.0e0
+GROW_DT              =   2.0e0
+MAX_INTEGRATOR_STEPS =   10000000000
+CFL_MAX              =   0.3
+NUM_INS_CYCLES       =   1
+NON_CONSERVATIVE_CONVECTIVE_OP_TYPE = "PPM"
+CONVECTIVE_FORM                     = "CONSERVATIVE"
+INIT_CONVECTIVE_TS_TYPE             = "FORWARD_EULER"
+CONVECTIVE_TS_TYPE                  = "FORWARD_EULER"
+NORMALIZE_PRESSURE   = TRUE
+VORTICITY_TAGGING    = FALSE
+TAG_BUFFER           = 2
+REGRID_CFL_INTERVAL  = 0.2
+DT_MAX               = 0.0005
+OUTPUT_U             = TRUE
+OUTPUT_P             = TRUE
+OUTPUT_F             = TRUE
+OUTPUT_OMEGA         = TRUE
+OUTPUT_DIV_U         = TRUE
+OUTPUT_RHO           = TRUE
+OUTPUT_MU            = TRUE
+RHO_IS_CONST         = FALSE
+MU_IS_CONST          = FALSE
+ERROR_ON_DT_CHANGE   = FALSE
+
+// Application
+CHI = RHO_S/DT_MAX
+PRECOND_REINIT_INTERVAL      = 1
+VC_INTERPOLATION_TYPE        = "VC_HARMONIC_INTERP"
+DENSITY_CONVECTIVE_LIMITER   = "CUI"
+VELOCITY_CONVECTIVE_LIMITER  = "CUI"
+DENSITY_TS                   = "SSPRK3"
+OPERATOR_SCALE_FACTORS       = 1.0
+EXPLICITLY_REMOVE_NULLSPACE  = FALSE
+ENABLE_LOGGING               = TRUE
+
+// IBFEMethod parameters
+SPLIT_FORCES                       = FALSE
+USE_JUMP_CONDITIONS                = FALSE
+USE_CONSISTENT_MASS_MATRIX         = TRUE
+IB_POINT_DENSITY                   = 1.0
+
```

Comment:
Please remove IBFE related options from the input file and all the rest extraneous options that are not used in this particular example.cpp. We need to clean this example as simple as possible.